tests/test_data.py

```diff
@@ -131,7 +131,7 @@ def test_query_metadata_earth_explorer(self, m):
     @requests_mock.Mocker(real_http=True)
     def test_query_metadata_srcid_scihub(self, m):
         m.post(
-            "https://scihub.copernicus.eu/apihub/search?format=json&rows=100&start=0",
+            "https://apihub.copernicus.eu/apihub/search?format=json&rows=100&start=0",

```

Comment:
Error found and resolved according to the suggestions. Test is also working fine now.

Reply:
When these are all, it is fine. Thanks!